apps/jobs/models.py

```diff
@@ -85,6 +91,12 @@ class Submission(TimeStampedModel):
     input_file = models.FileField(
         upload_to=RandomFileName("submission_files/submission_{id}")
     )
+    # Model to store submitted docker image information
+    docker_input_file = models.FileField(
```

Comment:
@savish28 this should be `submission_input_file`. We are going to store the output of docker evaluation in a separate file and not maintain multiple copies of input file. Having different submission flows isn't ideal.